.github/workflows/release.yml

```diff
@@ -249,3 +249,53 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           TAG: ${{ steps.create-release.outputs.computed-prefix }}${{ steps.create-release.outputs.version }}
+
+  check-plugin-version:
+    name: Check if plugin version exist in the registry and upload if not
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Check version
+        id: check-version
+        run: |
+          set -eo pipefail
+
+          snforge_scarb_plugin_version=$(grep version crates/snforge-scarb-plugin/Scarb.toml | cut -d '"' -f 2)
+          snforge_scarb_plugin=$(curl -s https://scarbs.xyz/api/v1/index/sn/fo/snforge_scarb_plugin.json | jq --arg version $snforge_scarb_plugin_version '[.[] | select(.v == $version)] | length > 0')
```

Comment:
Are you sure this curl works as intended? I'm getting:

```
$ curl -s https://scarbs.xyz/api/v1/index/sn/fo/snforge_scarb_plugin.json | jq --arg version 0.31.1 '[.[] | select(.v == $version)] | length > 0'
false

$ curl -s https://scarbs.xyz/api/v1/index/sn/fo/snforge_scarb_plugin.json | jq --arg version 31.31.1 '[.[] | select(.v == $version)] | length > 0'
false
```

So this returns false not matter if the version exists or not. Maybe I messed something though?

Reply:
It works, there's just no version `0.31.1` in the registry 😄

Reply:
This is what I am

![obraz](https://github.com/user-attachments/assets/2d3ca0e0-bf24-46ca-86f7-f210d8e02867)